### DISCUSSION TITLE
I'm not able to deploy Simple Storage to Sepolia on remix

### DISCUSSION THREAD

**Discussion by codex-scribe** [Cyfrin/foundry-full-course-cu#3125] [Q&A]
I am at the lesson where we are supposed to deploy our first smart contract to a testnet, and for some reason remix is not recognizing or filling in the account that I'm supposed to deploy from.  Here is the error that I see: 
![image](https://github.com/user-attachments/assets/b538309b-a76d-49d5-83cf-2a9636e9e445)

- [x] My metamask is connected to the the website and displays a green dot on the globe logo saying it is connected
- [x] Metamask is on sepolia testnet (not sepolia zksync or any other network)
- [x] My contract compiles
- [x] I was able to deploy to the Remix VM
- [x] I have SepoliaETH in my account (0.024)

The `account` field is blank, and it doesn't help to click any of those icons.  I also can't type into that field or copy-paste my address.

====

For searchability, the error says "Unfortunately it's not possible to create an account using the injected provider..."

**Comment by codex-scribe** [comment]
![image](https://github.com/user-attachments/assets/983a5a27-98f7-4d64-91ee-04f9a8bc9cb7)

I think I figured it out.  I had to go in the environments options menu (the plug icon next to `environment`.  I then chose the option that said Sepolia Testnet.  Then it worked.

**Reply by EngrPips** [reply]
Hello @codex-scribe, I am glad you figured it out.

**Reply by H-code9** [reply]
Hi @codex-scribe great to see you finding your way out, Best regards.

**Reply by EngrPips** [reply]
Indeed.
